src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -278,14 +283,14 @@ export const useGetTradeQuotes = () => {
 
   // cease fetching state when at least 1 response is available
   // more quotes will arrive after, which is intentional.
-  const isQuoteRequestFetching = useMemo(() => {
+  const isEverySwapperFetching = useMemo(() => {
     return (
       isDebouncing || isFetchingInput || combinedQuoteMeta.every(quoteMeta => quoteMeta.isFetching)
```

Comment:
Do we want to add a `combinedQuoteMeta.length` check here for paranoia? 
```suggestion
      isDebouncing || isFetchingInput || combinedQuoteMeta.every(quoteMeta => quoteMeta.isFetching)
```

Reply:
@woodenfurniture going to merge this as-is but mentioning for viz